### ISSUE TITLE
Unit Tests broken

### ISSUE THREAD

**Issue by AnalogJ** [balderdashy/sails-postgresql#89]
I'm attempting to validate and rebase my Array changes against the newest version of sails-postgresql and I noticed that many of the unit tests are referencing the `query.js` file which no longer exists.

I would be nice to have a working "state of the world" so we can test that our changes are compatible. I may have time to sit down sometime this week and update the unit tests if no one has done it already. 

-Jason

**Comment by sailsbot** [comment]
Thanks for posting, @AnalogJ.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
